platform/kube/controller.go

```diff
@@ -37,17 +37,16 @@ const (
 	NodeRegionLabel = "failure-domain.beta.kubernetes.io/region"
 	// NodeZoneLabel is the well-known label for kubernetes node zone
 	NodeZoneLabel = "failure-domain.beta.kubernetes.io/zone"
-	// IstioNamespace used by default for Istio cluster-wide installation
-	IstioNamespace = "istio-system"
 )
 
 // ControllerOptions stores the configurable attributes of a Controller.
 type ControllerOptions struct {
-	// Namespace to restrict controller to (empty to disable restriction)
-	Namespace    string
-	AppNamespace string
-	ResyncPeriod time.Duration
-	DomainSuffix string
+	// Namespace in which the controller runs. Can be empty if the controller does not run in a namespace
```

Comment:
2 points:

- "Can be empty if the controller does not run in a namespace" is that possible ? (controller outside of k8s ??)
- Is there a downside calling this IstioNamespace for clarity (vs the other one)

Reply:
Yes, for instance in rawVM, or in a standalone process in the computer.

We cannot call it istioNamespace, it is a generic field. "istio" prefix does not add much value, nor clarity.

Reply:
ok, thanks for the comment.

(semi rhetorical question: if it can be empty, what is it used for ? does it work with empty (there is a test for creating controller with empty namespace, yet it's used for ... ?) - ditto is there a test for empty vs non empty WatchedNamespace ?)